styleguide/components/31-molecules/display-switcher/README.md

```diff
@@ -1,2 +1,24 @@
 # Display switcher 
 
+## When to use this component
+
+Use the display switcher component to let users switch between display modes of one section of content.
```

Comment:
Current markup only allows switching between different **pages**, not between sections of content.

Reply:
The intention is that the display switcher is used to switch between display modes of one section of content. So the same content but shown in two different display modes, am I correct?

The display switcher could be used both when the URL does not change and when the URL does change, am I correct?

If this is the case, do you mean that the component should be changed so that it can support switching display modes without changing the URL? Should I make a JIRA ticket for this?

Reply:
In my view, you cannot use the same visual component to trigger a page reload or hide/display content on the current page.
The user will not know what to expect.

I believe this has already been discussed when validating the component in wireframes/design.

Reply:
@delrueba The fact that a page reload is done or content is hidden/displayed on the current page without page reload is rather something that is decided based on technical possibilities/limitations than decided based on what is the most wanted behavior, I am right? Because when it comes to switching display modes of the same content, for users it is actually rather strange that a page reload is needed for this.

Reply:
Yes, but we have tabs to toggle visibility of content on the same page.
These have a lot of extra markup for accessibility to indicate the roles as tab and tabpanel and javascript to actually make the thing work as intended, including keyboard handling etc.

This 'display switcher' was created precisely because of these technical limitations you speak of and to avoid confusion for the user. They are just plain links and cannot be used as tabs.

I agree that it's strange to trigger a page reload for this, but it's even stranger to have the same visual component sometimes trigger a page reload and sometimes not.